tests/QRamTest.qs

```diff
@@ -114,6 +114,151 @@
         Message("Test passed.");
     }
 
+    @Test("QuantumSimulator")
+    operation BBQRAMforTwoMemoryRegistersUnshuffled() : Unit {
+        let m0 = (0, true);
+        let m1 = (1, false);
```

Comment:
Sample data can be moved to helper function in this namespace and thus don't need duplicated.